### PR TITLE
PW-24 Implement 404 page

### PR DESCRIPTION
**Pull request by MCFreddie777**

### CODE REVIEW

**Review by Hemii — APPROVED** [review]
Approved ✔️

File: src/views/404.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+			404
+		</span>
+		<span class="text-bold text-2xl">
+			Ups! Nič sme nenašli...
+		</span>
+		<router-link
+			to="/"
+			class="bg-primary hover:bg-accent font-bold text-white rounded mt-5 py-2 px-4 cursor-pointer"
+		>
+			Ísť na domov
```
Comment (KToldy) [inline src/views/404.vue:13]: "Ísť na domov" by som zmenil asi len na "domov" to ísť mi tam dáko vadí ale inak super, nie je čo vytknúť

Comment (MCFreddie777) [reply src/views/404.vue:13]: Nitpick, don't you think? 😄 

Also, 🇬🇧   please.

Comment (KToldy) [reply src/views/404.vue:13]: Yeah it's very subjective

**Review by KToldy — APPROVED** [review]

**Pull request merged**